lib/cipherstash/client/rpc.rb

```diff
@@ -270,17 +270,23 @@ def put_stream(collection, records)
         # isn't the most idiomatic code on earth...
         requests = [[begin_request], records].lazy.flat_map { |x| x }
 
-        res = @metrics.measure_rpc_call("putStream") do
-          stub.put_stream(requests, metadata: rpc_headers)
-        end
+        num_inserted = 0
+        @logger.info("Cipherstash::Client::RPC#put_stream") { "Start streaming upsert of documents..." }
+
+        @metrics.measure_rpc_call("putStream") do
+          stub.put_stream_bi_direction(requests, metadata: rpc_headers).each do |res|
+            unless res.is_a?(Documents::StreamingPutReply)
+              raise Error::StreamingPutFailure, "expected Documents::StreamingPutReply response, got #{res.class} instead"
+            end
+            @logger.info("Cipherstash::Client::RPC#put_stream") { "#{res.numInserted} records inserted." }
```

Comment:
```suggestion
            @logger.debug("Cipherstash::Client::RPC#put_stream") { "#{res.numInserted} records inserted." }
```
Same comment here about using debug instead of info.